utils/lib/connection_database.sql

```diff
@@ -51,7 +51,17 @@ CREATE TABLE site_type(
   name TEXT
 );
 
--- Tile table, contains type and name of tile and location in grid.
+-- Tile table, contains type and name of tile and location in the physical grid.
+CREATE TABLE phy_tile(
+  pkey INTEGER PRIMARY KEY,
+  name TEXT,
+  tile_type_pkey INT,
```

Comment:
I just want to talk through the mapping.   Upon a tile split, one physical location (phy_tile.pkey) will be presented by multiple VPR tiles (tile.pkey).  Therefore the tile table should have the foreign key, rather than phy_tile?  Yes?

Reply:
It works like that: First the prjxray grid is imported to the `phy_tile` table. Then the mapping physical <-> VPR is generated. Once done, entires from the `phy_tile` table are "copied" to the `tile` table with modified locations and the same pkey (actually didn't think if the pkey needs to stay the same).

The correspondence between these two tile tables is held in the `grid_loc_map` table. It is done by keeping location map rather than direct `tile` <-> `phy_tile` via foreign key.

Reply:
So the conclusion is that I will go with the foreign keys.